pcsx2/Pcsx2Config.cpp

```diff
@@ -237,7 +237,7 @@ void Pcsx2Config::GSOptions::LoadSave( IniInterface& ini )
 
 int Pcsx2Config::GSOptions::GetVsync() const
 {
-	if (g_LimiterMode == Limit_Turbo)
+	if (g_LimiterMode == Limit_Turbo || !FrameLimitEnable)
```

Comment:
Is it still possible to use Vsync without framelimiter ?

Reply:
Every time you disable FL, vsync gets nuked now. 
Which is [bad](2053) - albeit certainly at least consistent now.

Reply:
@turtleli you can merge it. And we can see later how to handle framelimiter/vsync case (if there is really an issue).

Reply:
Ok, I'll merge after Travis CI is fixed.